packages/media-utils/src/utils/upload-media.ts

```diff
@@ -134,9 +134,16 @@ export function uploadMedia( {
 			// Reset to empty on failure.
 			setAndUpdateFiles( index, null );
 
-			let message;
-			if ( error instanceof Error ) {
-				message = error.message;
+			let message: string;
```

Comment:
This change was to get the error showing correctly, should double check this is required.

Reply:
yep; without this change, the generic error is shown:

![image](https://github.com/user-attachments/assets/0c1529f6-7b28-4d5c-9cf7-64144c560212)

Testable in https://playground.wordpress.net/?gutenberg-pr=69125